modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/LocalHealthcheck.scala

```diff
@@ -31,7 +31,10 @@ object LocalHealthcheck {
     mkPeersR.map(make(_, retryPolicy, nodeClient, clusterStorage))
   }
 
-  case class PeerUnresponsive(id: PeerId) extends NoStackTrace
+  case class PeerUnresponsive(id: PeerId) extends NoStackTrace {
+    implicit val show = PeerId.show
+    override val getMessage = id.show
```

Comment:
Regarding `implicit val show = PeerId.show` maybe you can pass the show instance explicitly to `show` function? 🤔

Reply:
@TheMMaciek I don't think `.show` takes an implicit arg. Is there a way to do it that I'm not seeing?

Also, thank you for the comment -- that's the wrong implicit; I wanted `shortShow`